4_communicating_results/retrospective_of_4_communicating_results.md

```diff
@@ -1,19 +1,22 @@
 
 ## Introduction
- 
+Date: 19 February - 11 March, 2024
+This milestone focuses on the Communicating Results phase of the project.
 
 ## Acheivements
- 
+1- We worked on fixing our repositry's main structure.
+2- We worked on creating our Artifact
+3- We worked on describing and identifing our target audience
+4- We worked on the script for our final presentation
+5- We worked on finalizing the ChatGPT-simulation analysis
 
 ## Challenges Faced
- 
-
-## Tools and Technique Evaluation
- 
+NA 
```

Comment:
Coming up with the artifact was a big challenge that required a lot of brainstorming.
We faced problems in finding a time where enough people in the team was available to meet.

Reply:
Fixed the changes